intelmq/bots/parsers/turris/parser.py

```diff
@@ -1,50 +1,49 @@
-# SPDX-FileCopyrightText: 2015 robcza
+# SPDX-FileCopyrightText: 2023 Filip Pokorný
 #
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
-# -*- coding: utf-8 -*-
-import csv
-import io
-
-from intelmq.lib import utils
 from intelmq.lib.bot import ParserBot
 
+PORTS = {
+    "ftp": 21,
+    "telnet": 23,
+    "http": 80
+    # smtp uses both 25 and 587, therefore we can't say for certain
```

Comment:
is it guaranteed that the port can't be 990 for FTP over TLS (FTPS) or 443 for HTTP over TLS (HTTPS)?